osquery/CMakeLists.txt

```diff
@@ -109,6 +109,8 @@ if(WINDOWS)
   ADD_OSQUERY_LINK_CORE("Wtsapi32.lib")
   ADD_OSQUERY_LINK_CORE("wbemuuid.lib")
   ADD_OSQUERY_LINK_CORE("taskschd.lib")
+  ADD_OSQUERY_LINK_CORE("dbghelp.lib")
```

Comment:
Just an aside - before we kick off tests on our build hosts, we'll want to build a minimal chocolatey package for some of the Windows SDK libs, as I don't think we're interested in installing the full windows SDK on the build hosts. Will investigate and get something together soon!

Reply:
libs shipped, this is all set.